### DISCUSSION TITLE
Blazor 6.0 Calling PersonExistCommand from PersonEdit business object

### DISCUSSION THREAD

**Discussion by jhuerta3** [MarimerLLC/csla#3098] [Questions] [answered]
Hello All,

Can someone share how we call the Person Exist from within the PesonEdit object. I tried the following but it seems this is no longer the correct way to call it. 

        public static bool Exists(Guid id)
        {
            var cmd = DataPortal.Create<PersonExistsCommand>(id);
            cmd = DataPortal.Execute(cmd);
            return cmd.PersonExists;
        }
Thank you.

**Comment by theafricaninternetgroup** [comment, marked as answer]
```
public  bool exists(Guid id)
{
    var portal=ApplicationContext.GetRequiredServices<IDataportal<PersonExistsCommand>>();
var cmd=portal.Create(id);
cmd=portal.Execute(cmd);
return cmd.PersonExists;
}
````

**Reply by theafricaninternetgroup** [reply]
Try this

**Reply by jhuerta3** [reply]
thank you @theafricaninternetgroup I will try it.

**Comment by jhuerta3** [comment]
I figured I would share this option also. 

        public bool Exists(string email, [Inject] DataAccess.IEmployeeDal dal)
        {
            var cmd = dal.Exists(email);
            return cmd;
        }
